faith_presentation/src/commonMain/kotlin/net/thechance/mena/faith/presentation/extensions/timeFormatingHelper/TimeUnit.kt

```diff
@@ -0,0 +1,5 @@
+package net.thechance.mena.faith.presentation.extensions.timeFormatingHelper
+
+enum class TimeUnit {
+    SECONDS, MINUTES, HOURS, DAYS, WEEKS, MONTHS, YEARS
```

Comment:
It would be easier to read if it's separated by lines.
```suggestion
    SECONDS, 
    MINUTES, 
    HOURS, 
    DAYS, 
    WEEKS, 
    MONTHS, 
    YEARS
```